source/framework/core/src/TRestAnalysisPlot.cxx

```diff
@@ -562,6 +568,9 @@ TRestAnalysisTree* TRestAnalysisPlot::GetTree(TString fileName) {
 }
 
 TRestRun* TRestAnalysisPlot::GetRunInfo(TString fileName) {
+    fRun->OpenInputFile(fileName);
```

Comment:
This doesn't looks fine to me.

We can consider to move to singleton:
https://stackoverflow.com/questions/1008019/c-singleton-design-pattern

But I am afraid it would require a lot of changes.

Reply:
Ok, I am not familiar with. 

what I am worried now is that pandaxiii pipeline did not succeed.